source/documentation/monitoring_apps/metrics.md

```diff
@@ -35,13 +66,13 @@ To set up the metrics app:
 
 	You can also set environment variables by amending the manifest file. We recommend that you use this method for optional environment variables that do not contain secret information. Refer to the [https://github.com/alphagov/paas-metric-exporter](https://github.com/alphagov/paas-metric-exporter) repository for more information.
 
-4. Start your app by running `cf start metric-exporter`.
+1. Start the metrics exporter app by running `cf start metric-exporter`.
 
 You can now check your monitoring system to see if you are receiving metrics.
 
 ### Troubleshooting
 
-If you are not receiving the metrics, check the [logs](/monitoring_apps.html#logs) for the metrics exporter app. If you still need help, please contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+If you are not receiving the metrics, check the Prometheus or metrics exporter app [logs](/monitoring_apps.html#logs). If you still need help, contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
```

Comment:
I don't think that's true. This section is only relevant to `paas-metric-exporter` app itself. Our tenants have no access to the prometheus endpoint logs. If the URL is down, it's something we need to investigate and fix as a team and over time may be crucial part of the platform.